tracim/development.ini.base

```diff
@@ -235,6 +237,25 @@ email.reply.use_txt_parsing = true
 # it's just an empty file use to prevent concurrent access to imap unseen mail
 email.reply.lockfile_path = %(here)s/email_fetcher.lock
 
+## Jitsi_meet_integration
+jitsi_meet.activated = False
+# Domain/IP where jitsi_meet is located, example: "meet.jit.si", "127.0.0.1"
+jitsi_meet.domain = your_jitsi_meet_instance
+# Choose between token and anonymous auth, anonymous is default one.
+# for token auth config, see : https://github.com/jitsi/lib-jitsi-meet/blob/52eb3decf6542413c739ce2209456fac728a89d5/doc/tokens.md
+jitsi_meet.use_token = False
+# If token mode is choosen, you need to provide a token generator.
+# "local" value mean that tracim himself will create token for jitsi-meet
```

Comment:
ok for local case. What is/are the other possibility for token_generator if not local ?

Reply:
Support for distant token generator is not added yet but it should be possible later. I complete the .ini about this.